homeassistant/components/landisgyr_heat_meter/const.py

```diff
@@ -0,0 +1,181 @@
+"""Constants for the Landis+Gyr Heat Meter integration."""
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import TEMP_CELSIUS
+from homeassistant.helpers.entity import EntityCategory
+
+DOMAIN = "landisgyr_heat_meter"
+
+HEAT_METER_SENSOR_TYPES = (
+    SensorEntityDescription(
+        key="heat_usage_gj",
+        icon="mdi:fire",
+        name="Heat usage GJ",
```

Comment:
I don't think we should include the unit in the sensor name. Same comment for other sensors.

Reply:
I've let theses as they were, even though I understand what you mean, I'll have to distinquish between GJ and MWh in the names at least now.

Reply:
Done this as well.